elide-core/src/main/java/com/yahoo/elide/core/filter/Operator.java

```diff
@@ -450,7 +450,19 @@ private static <T> Predicate<T> notinfix(Path fieldPath, List<Object> values,
     //
     // Null checking
     private static <T> Predicate<T> isNull(Path fieldPath, RequestScope requestScope) {
-        return (T entity) -> getFieldValue(entity, fieldPath, requestScope) == null;
+        return (T entity) -> {
+            Object val = getFieldValue(entity, fieldPath, requestScope, true);
+            if (val == null) {
+                return true;
+            } else if (val instanceof Collection<?> collection) {
+                for (Object item : collection) {
+                    if (item == null) {
```

Comment:
This feels more like hasNull or hasMember(null) instead of isNull.   The behavior might be unexpected.  Can we accomplish the same thing with hasMember(null)?